test/tests/functional/pbs_sister_mom_crash.py

```diff
@@ -53,7 +53,8 @@ def test_sister_mom_crash(self):
             self.skipTest("test requires atleast two MoMs as input, " +
                           "use -p moms=<mom1:mom2>")
         sister_mom = self.moms.keys()[1]
-        script = "pbsdsh dd if=/dev/zero of=/dev/null"
+        pbsdsh_path = self.du.which(exe='pbsdsh')
+        script = "%s dd if=/dev/zero of=/dev/null" % pbsdsh_path
```

Comment:
The problem with which() is that it mainly looks through the path to find a program.  We know where pbsdsh is.  Just join pbs_conf['PBS_EXEC'] with bin and pbsdsh